src/components/UseOfFunds/FundsContent.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+	<div class="use-of-funds">
+		<div class="use-of-funds-section">
+			<div class="use-of-funds-section-intro">
+				<h2>{{ content.intro.headline }}<span v-if="content.provisional !== ''">*</span></h2>
+				<div>{{ content.intro.text }}</div>
+			</div>
+		</div>
+
+		<FundsDistributionAccordion :application-of-funds-data="content.applicationOfFundsData" />
+
+		<div class="use-of-funds-section">
+			<p class="use-of-funds-info-text" v-html="content.detailedReports.mixed.text"></p>
+		</div>
+		<div class="use-of-funds-section use-of-funds-section-two-cols">
+			<div class="use-of-funds-column">
+				<div class="use-of-funds-benefits-list">
+					<h2>{{ content.benefitsList.headline }}</h2>
+					<ul class="use-of-funds-icon-list">
+						<li
+							v-for="benefit in content.benefitsList.benefits"
+							:class="'use-of-funds-icon-list-item-' + benefit.icon"
+							:key=benefit.text
+						>
+							{{ benefit.text }}
+						</li>
+					</ul>
+				</div>
+			</div>
+			<div class="use-of-funds-column">
+				<div class="use-of-funds-comparison">
+					<h2>{{ content.comparison.headline }}</h2>
+					<div>
+						<p v-for="text in content.comparison.paragraphs" :key="text">{{ text }}</p>
+						<h3>{{ content.comparison.subhead }}</h3>
+					</div>
+					<CompanyBudgets
+						:companies="content.comparison.companies"
+						:citation-label="content.comparison.citationLabel"
+					/>
+				</div>
+			</div>
+		</div>
+		<div class="use-of-funds-section use-of-funds-section-orgchart">
+			<div class="use-of-funds-orgchart-text">
+				<h2>{{ content.orgchart.headline }}</h2>
+				<div>
+					<p v-for="( paragraph, idx ) in orgChartParagraphs" :key="idx" v-html="paragraph"></p>
+				</div>
+			</div>
+			<div class="use-of-funds-orgchart-image">
+				<img :src="content.orgchart.imageUrl"  :alt="content.orgchart.headline"/>
+			</div>
+		</div>
+		<div class="banner_model-section use-of-funds-section-call-to-action">
+			<button class="use-of-funds-button" onclick="location.href='/'">{{ content.callToAction }}</button>
```

Comment:
This has the wrong handler (`onclick` vs `@click`) and should close the UoF modal instead of going home.